### PR TITLE
Hmis 203 add code validation

### PR DESCRIPTION
**Pull request by karanotts**
Added pester script to lint and validate terraform files.

### CODE REVIEW

File: ci/terraform-lint.tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+$RootPath = Split-Path -Parent $MyInvocation.MyCommand.Path | Split-Path
```
Comment (santhoshhmcts) [inline ci/terraform-lint.tests.ps1:2]: Kara can you give me explain what this code is trying to achieve.

File: infrastructure/terraform/05-inputs-api-policies.tf [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+      operationId  = "Request-Hearing"
+      format       = "rawxml-link"
+      templateFile = "api-op-request-hearing-policy.xml"
     },
-        { 
-      operationId     = "Health-Check"
-      format          = "rawxml-link"
-      templateFile    = "api-op-health-check-policy.xml"
+    {
+      operationId  = "Health-Check"
```
Comment (subrotomukherjee) [inline infrastructure/terraform/05-inputs-api-policies.tf:9]: Operation Id is incorrect here.
The correct one is: health-check

File: infrastructure/terraform/05-inputs-api-policies.tf [hunk 3]
```diff
@@ -1,14 +1,14 @@
 variable "api_policies" {
-    default = [
-    { 
-      operationId     = "Request-Hearing"
-      format          = "rawxml-link"
-      templateFile    = "api-op-request-hearing-policy.xml"
+  default = [
+    {
+      operationId  = "Request-Hearing"
```
Comment (subrotomukherjee) [inline infrastructure/terraform/05-inputs-api-policies.tf:4]: Operation Id is incorrect here.
The correct one is: request-hearing

File: azure-pipelines.yml [hunk 4]
```diff
@@ -20,10 +20,19 @@ variables:
 
 stages:
 
+  - stage: CI
+    displayName: Validate the code
```
Comment (santhoshhmcts) [inline azure-pipelines.yml:24]: validate the ci/cd code perhaps?

File: ci/terraform-lint.tests.ps1 (hunk 1 above)
Comment (karanotts) [reply ci/terraform-lint.tests.ps1:2]: https://ss64.com/ps/syntax-myinvocation.html

**Pull request merged**